tutor/specs/models/task-plans/teacher/tasking.spec.js

```diff
@@ -19,20 +19,20 @@ describe('Teacher tasking plan tasking', () => {
 
     tasking.due_at = '2016-10-14T12:00:00.000Z';
     tasking.setOpensDate('2016-10-20T12:00:00.000Z');
-    expect(tasking.opens_at).toEqual('2016-10-14T12:00:00.000Z');
+    expect(tasking.opens_at).toEqual('2016-10-14T11:59:00.000Z');
 
     tasking.setOpensTime('2016-10-20T19:42:00.000Z');
-    expect(tasking.opens_at).toEqual('2016-10-14T12:00:00.000Z');
+    expect(tasking.opens_at).toEqual('2016-10-14T11:42:00.000Z');
```

Comment:
This seems odd... how does 19:42 become 11:42? Due date does not end in :42 either.